test/e2e/performanceprofile/functests/5_latency_testing/5_latency_testing_suite_test.go

```diff
@@ -68,7 +69,12 @@ var _ = BeforeSuite(func() {
 			if err != nil {
 				testlog.Error("could not update the profile with the desired CPUs sets")
 			}
+		} else {
+			testlog.Info("current performance profile configuration is valid for the suite, no need to reconfigure it")
 		}
+	} else {
+		testlog.Warning("found 0 nodes with sufficient cpus for the performance profile configuration. Quiting ..")
+		os.Exit(1)
```

Comment:
never use `os.Exit` in tests please. better to fail OR to check the prerequisites earlier and act accordingly
Additionally, can we avoid `else`s? :)

Reply:
thanks for pointing that out, I'll verify and update the code accordingly.
 here the suite did not update the cluster configuration (yet), any reason why not using os.exit in tests?